content/blog/2016/2016-10-11-stage-lock-milestone.adoc

```diff
@@ -74,7 +74,7 @@ lock("myResource"){
 }
 ----
 
-=== milestone
+=== Milestone
 
 The `milestone` step is the last piece of the puzzle to replace functionality originally intended for `stage` and adds even more control for handling concurrent builds of a job. The `lock` step limits the number of builds running concurrently in a section of your Pipeline while the `milestone` step ensures that older builds of a job will not overwrite or clobber a newer build.
 
```

Comment:
How are `milestone` calls matched up?  What happens if I have the following:

```groovy
// returns a value between 1 and 10
def a = getData()

milestone
if (a > 5) {
    milestone
}

milestone
```

Are the milestones simply identified by increment or somehow by line number/location?